packages/sacado/src/Sacado_ConfigDefs.h

```diff
@@ -119,14 +119,12 @@ Questions? Contact David M. Gay (dmgay@sandia.gov) or Eric T. Phipps
 /* Determine if the new fad design is supported.  Requies C++11,
    and if gcc, version 4.8 or greater.
 */
-#if defined(HAVE_SACADO_CXX11)
-#  if defined(__GNUC__) && !defined(__clang__)
-#    if (__GNUC__ > 4) || ((__GNUC__ == 4) && (__GNUC_MINOR__ >= 8) )
-#      define SACADO_ENABLE_NEW_DESIGN 1
-#    endif
-#  else
+#if defined(__GNUC__) && !defined(__clang__)
+#  if (__GNUC__ > 4) || ((__GNUC__ == 4) && (__GNUC_MINOR__ >= 8) )
 #    define SACADO_ENABLE_NEW_DESIGN 1
 #  endif
+#else
+#  define SACADO_ENABLE_NEW_DESIGN 1
```

Comment:
@etphipp I guess this could also be removed, since C++17 requires a much more recent gcc version than 4.8 :smiley:

Reply:
And so the question would be: if so, should we also get rid of the "Sacado old design" ? :smiley:

Reply:
(of course that would be another PR)

Reply:
This has been on the to-do list for a while.  I think some codes might still be using the old design though.

Reply:
Ok, not to be done now.